### ISSUE TITLE
Exception calling "Save" with "0" argument(s): "Error saving file

### ISSUE THREAD

**Issue by namnamir** [dfinke/ImportExcel#845]
I previously posted about this issue on [#687](https://github.com/dfinke/ImportExcel/issues/687).

Nobody could help to fix the issue. I have tried to debug it myself. There is a bug in the code and I hope it gets fixed in the next release :)

## What is the problem?
When I tried to save a sheet in the `.xlsx` file, sometimes it showed me an error and sometimes not. The confusing part of it was arbitrary reaction of the code. The error code was something list this:
```
Exception calling "**Save" with "0" argument(s)**: "Error saving file
Close-ExcelPackage.ps1:27 char:29
 $ExcelPackage.Save()
```
## How has it gotten fixed?
By doing the reverse engineering and _trial and error_ method, I found the issue. The problem comes from **[Add-ExcelTable](https://github.com/dfinke/ImportExcel/blob/481e2c6cc897b41bf26fd177d9503a1179546d2c/mdHelp/en/Add-ExcelTable.md)**. It cannot parse it while the _first row (header)_ of the table has **duplicated names**. In my case, I wrote a simple check to assure that the first row doesn't contain any duplication. Here is a very simple solution:

```powershell
# an array to save the name of the properties
$Temp_Name = @()

# iterate over properties
foreach ($PN in $Properties.Keys) {
       [...]

        # if "Add-ExcelTable" is used, it cannot handle any duplication as the header of the table.
        if ($Properties.$PN.Display_Name -in $Temp_Name) {
            # add a random number to the duplicated header name
            $Properties.$PN.Display_Name = $Properties.$PN.Display_Name + " " + ((10..99) | Get-Random -Count 1)
            # add a random number to the duplicated header name
            $Temp_Name += $Properties.$PN.Display_Name
        } # if
        else {
            $Temp_Name += $Properties.$PN.Display_Name
        } # else

       [...]
} # foreach
```
I hope it can solve the issue of others.

**Comment by ChrisLegend95** [comment]
@namnamir 
I found the same problem with Export-Excel

And I found out when I was using PS ISE I finger out why it did not work to save as I did before. I try to change it back to Export-CSV and it told me I did not have access to save to my C:\. 

What I did after it was I save my script and close the PS ISE and run it again but as an Admin... and now it works fine to Export-Excel.

I hope this will help you all.

//Chris
